detections/endpoint/windows_executable_masquerading_as_benign_file_types.yml

```
@@ -0,0 +1,64 @@
+name: Windows Executable Masquerading as Benign File Types
+id: 0470c8e7-dd8d-420f-8302-073e8a2b66f0
+version: 1
+date: '2025-11-20'
+author: Teoderick Contreras, Splunk
+status: production
+type: TTP
```

I suggest we keep this to an anomaly until we get further data from the field just to play it safe. Because typically there are uncommon extensions in the wild.
```suggestion
type: Anomaly
```